js/index.js

```diff
@@ -12,7 +104,65 @@ fetch("https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&hou
     } else {
         console.log(report);
     } 
+    //Displaying reports
+    const city = document.querySelector('.city');
+    city.innerHTML = report.timezone;
+    const hourlyTemp = document.querySelector('.grid');
+    hourlyTemp.innerHTML = (report.hourly.temperature_2m).join(" ");
```

Comment:
For readability, I would recommend having these listed along with the hour they pertain to. For example, something like
1:00 25.3 C
2:00 24.8 C
and so on.